README.adoc

```diff
@@ -42,37 +45,50 @@ and configure a _Run/Debug configuration_ like this:
+
+This is an open source project, and you are more than welcome to contribute :heart:!
 
-If you're just starting out with OptaPlanner and want to contribute,
+
+* If you're just starting out with OptaPlanner and want to contribute,
 take a look at our https://issues.redhat.com/issues/?jql=project%20%3D%20PLANNER%20AND%20status%20in%20(Open%2C%20Reopened)%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC[starter issues].
 They're specifically chosen to be easier for first time contributors.
 
-== Developing Drools, OptaPlanner and jBPM
+* If you want to contribute or start an opinionated discussion, join our https://groups.google.com/g/optaplanner-dev[discussion] or send an e-mail directly to optaplanner-dev@googlegroups.com.
+
+* If you want to submit an issue, check out the https://issues.redhat.com/projects/PLANNER/issues[OptaPlanner Jira project].
 
-*If you want to build or contribute to a kiegroup project, https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/README.md[read this document].*
+=== Time to make a change?
 
-*It will save you and us a lot of time by setting up your development environment correctly.*
-It solves all known pitfalls that can disrupt your development.
-It also describes all guidelines, tips and tricks.
-If you want your pull requests (or patches) to be merged into main, please respect those guidelines.
+Every change must go through *Pull Request*. Please follow the <<Contributing>> section for more details. OptaPlanner has a CI in place that runs checks against your branch after you submit the PR. Don't worry if something gets wrong or breaks other parts of functionality; our friendly PR maintainers will help you with that.
 
-=== Code style
+=== Contributing
+
+1. Fork it (https://github.com/kiegroup/optaplanner)
+2. Create your feature branch (`git checkout -b feature`)
+3. Commit your changes (`git commit -am 'Add some feature'`)
+4. Push to the branch (`git push origin feature`)
+5. Create a new Pull Request
+
+=== Code standards
+
+Your code is automatically formatted according to the _Import and Code Style_ conventions during every Maven build. CI checks enforce those conventions too, so be sure to build your project with maven before creating your PR:
+----
+mvn clean install
+----
+For information about how to setup code style checks, see https://github.com/kiegroup/optaplanner/blob/main/build/optaplanner-ide-config/ide-configuration.adoc[IDE Setup Instructions].
 
-OptaPlanner has adopted the https://github.com/quarkusio/quarkus[Quarkus] code style, enforces it, and automatically formats code during the build.
-To setup your IDE, please see the
-<<build/optaplanner-ide-config/ide-configuration.adoc#, IDE Setup Instructions>>.
+=== Building your OptaPlanner project
 
-=== Definition of Done
+Use one of the following ways to build your OptaPlanner project:
+- :rocket: *build-fast*: `mvn clean install -Dquickly` skips any checks and code analysis (~1 min)
```

Comment:
The list is broken (see https://github.com/dupliaka/optaplanner/blob/PLANNER-2684/README.adoc#building-your-optaplanner-project). Probably a blank line is needed before the first list item.

Reply:
Thanks for noticing! Fixed that, rebased and ready to merge